src/Draco.Compiler/Internal/Syntax/Parser.cs

```diff
@@ -1546,6 +1645,8 @@ private SeparatedSyntaxList<TNode> ParseSeparatedSyntaxList<TNode>(
 
     private SyntaxToken? ParseVisibilityModifier() => IsVisibilityModifier(this.PeekKind()) ? this.Advance() : null;
 
+    private SyntaxToken? ParseGlobalModifier() => this.PeekKind() == TokenKind.KeywordGlobal ? this.Advance() : null;
```

Comment:
Global isn't even lexed

Reply:
I feel that's like the 4th time I re-add it...